securedrop_export/export.py

```diff
@@ -120,48 +154,43 @@ def exit_gracefully(self, msg, e=False):
         since non-zero exit values will cause system to try alternative
         solutions for mimetype handling, which we want to avoid.
         """
-        sys.stderr.write(msg)
-        sys.stderr.write("\n")
         logger.info('Exiting with message: {}'.format(msg))
-        if e:
+        if not e:
+            sys.stderr.write(msg)
+            sys.stderr.write("\n")
+        else:
```

Comment:
:+1:  this should fix the bug where we were seeing extra output in our error status code (will test after code review)